crates/invoker-impl/src/options.rs

```diff
@@ -71,12 +71,12 @@ pub struct Options {
 
     /// # Message size warning
     ///
-    /// Threshold to log a warning in case protocol messages coming from service endpoint are larger than the specified amount.
+    /// Threshold to log a warning in case protocol messages coming from service are larger than the specified amount.
```

Comment:
from a service

Reply:
This is a case where using the term service endpoint was perhaps fine...